feconf.py

```diff
@@ -185,6 +185,9 @@
 # limit applies to both the explorations playlist and the collections playlist.
 MAX_LEARNER_PLAYLIST_ACTIVITY_COUNT = 10
 
+# The maximum number of goals allowed in the learner goals of the learner.
+MAX_CURRENT_GOALS_COUNT = 10
```

Comment:
I thought it was 5? I'm happy to go up to 8 but more than that is a bit excessive IMO.

Reply:
Updated to 5. PTAL!